packages/jaeger-ui/src/model/transform-trace-data.test.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { orderTags, deduplicateTags } from './transform-trace-data';
+
+describe('orderTags()', () => {
+  it('correctly orders tags', () => {
+    const orderedTags = orderTags(
+      [
```

Comment:
If you could expand the test data and expected output a bit to cover that it sorts within a tag prefix (i.e.: `'http.a'` is before `'http.z'`) and that it respects the order of `topTagPrefixes` (i.e.: if `topTagPrefixes` is `['z.', 'a.', 'http.']` then the result will have this order), that would be great.


I think this would do it:
```
const orderedTags = orderTags(
  [
    { key: 'b.ip', value: '8.8.4.4' },
    { key: 'http.status_code', value: '200' },
    { key: 'a.ip', value: '8.8.8.8' },
    { key: 'z.ip', value: '8.8.8.16' },
    { key: 'http.message', value: 'ok' },
  ],
  ['z.', 'a.', 'http.']
)
expect(orderedTags).toEqual([
  { key: 'z.ip', value: '8.8.8.16' },
  { key: 'a.ip', value: '8.8.8.8' },
  { key: 'http.message', value: 'ok' },
  { key: 'http.status_code', value: '200' },
  { key: 'b.ip', value: '8.8.4.4' },
]);
```